### ISSUE TITLE
[Feature Request] Add Support for ca-west-1 region

### ISSUE THREAD

**Issue by allank-fd** [DataDog/datadog-lambda-python#420]
## Expected Behavior
I should be able to use lambda runtime layer in ca-west1 region of AWS

## Actual Behavior
```
╷
│ Error: creating Lambda Function (xxxxxx): operation error Lambda: CreateFunction, https response error StatusCode: 403, RequestID: c0649569-c7d1-44e5-900b-d1b925b6af02, api error AccessDeniedException: User: arn:aws:sts::xxxxx is not authorized to perform: lambda:GetLayerVersion on resource: arn:aws:lambda:ca-west-1:464622532012:layer:Datadog-Python39:78 because no resource-based policy allows the lambda:GetLayerVersion action
```

## Steps to Reproduce the Problem

  1. Attempt to use Datadog-Python39:88  on lambda
  2. Apply Terraform

## Specifications

  - Datadog Lambda Layer version: 88
  - Python version: 3.9
  
## Stacktrace
  
  ```
An error occurred (AccessDeniedException) when calling the GetLayerVersion operation: User: arn:aws:sts::xxxxxx is not authorized to perform: lambda:GetLayerVersion on resource: arn:aws:lambda:ca-west-1:464622532012:layer:Datadog-Python39:88 because no resource-based policy allows the lambda:GetLayerVersion action
  ```

**Comment by astuyve** [comment]
Hi @allank-fd, thanks for reaching out! That layer doesn't exist because we don't support ca-west-1 yet. I'll leave this open as a feature request.

For now, please feel free to use this library directly from pypi in that region.

Thanks!

**Comment by astuyve** [comment]
Quick update here, we're nearly ready to support this. I have one or two minor bugs to fix in our ci/cd process but I feel like I should have a good handle on them by next week.

Thanks!

**Comment by astuyve** [comment]
this is released in [v5.90.0](https://github.com/DataDog/datadog-lambda-python/releases/tag/v5.90.0)

**Comment by allank-fd** [comment]
thanks @astuyve , we'll get this deployed
